src/core/quicdef.h

```diff
@@ -376,6 +376,12 @@ CXPLAT_STATIC_ASSERT(
 //
 #define QUIC_SEND_PACING_INTERVAL               1000
 
+//
+// The maximum number of MTUs (datagram payload length) we will allow the
```

Comment:
It's more subtle than this. Might be safest to say "Congestion window increase limit L from RFC 3465".

Reply:
I'd rather have a complete explanation here instead of requiring someone to go look at an RFC. I'm fine referencing an RFC, but I don't want to require knowledge of an RFC.

Reply:
How about this then: "Slow start congestion window increase limit L from RFC 3465, to reduce burstiness". That's at least a complete description of how the threshold is used.